maven-plugin/src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -319,7 +319,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         List<File> foundFiles = new ArrayList<>();
         for (String filename : ds.getIncludedFiles()) {
-            foundFiles.add(new File(newBasedir, filename));
+            File foundFile = new File(newBasedir, filename);
+            if(foundFile.canWrite()){
+                foundFiles.add(foundFile);
+            }
```

Comment:
@fbyrne However, this fixes it before it ever becomes an issue.  Given the cost of creating objects in loop and fact that we are already doing this in the other location, I think it would be cheaper to just of the first fix.  Thoughts?